app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala

```diff
@@ -195,14 +196,16 @@ class ApplicationService @Inject() (
     }
   }
 
-  def findAndRecordServerTokenUsage(serverToken: String): Future[Option[ApplicationWithSubscriptions]] = {
+  def findAndRecordServerTokenUsage(serverToken: String): Future[Option[(ApplicationWithSubscriptions, String)]] = {
     timeFuture("Service Find And Record Server Token Usage", "application.service.findAndRecordServerTokenUsage") {
       (
         for {
           app           <- OptionT(applicationRepository.findAndRecordServerTokenUsage(serverToken))
+          // these are identical otherwise the find would not work
+          // serverToken    = app.tokens.production.accessToken
```

Comment:
probably worth removing the commented out code

Reply:
I've improved the comment. It's meant to prevent a misunderstanding in future.